subprojects/docs/src/docs/userguide/working_with_files.adoc

```diff
@@ -334,14 +334,17 @@ See how we're using the `compileJava` task as the source of the files to package
 Using a task directly as an argument like this relies on it having <<more_about_tasks.adoc#sec:task_inputs_outputs,defined outputs>>, so it won't always be possible. In addition, this example could be improved further by relying on the Java plugin's convention for `destinationDir` rather than overriding it, but it does demonstrate the use of project properties.
 
 [[sec:single_file_paths]]
-=== Single file paths
+=== Single file
```

Comment:
I'm OK with this change, but it should then really be "Single files", i.e. plural. The next section is _File collections_ after all.

Reply:
I'm also tempted to explicitly mention "directories" here as I think it's natural to think of them as distinct from files. "Single files & directories"?

Reply:
Yep, makes sense